bin/worker

```diff
@@ -1,8 +1,4 @@
 #!/usr/local/bin/ruby
-if ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'] && !ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'].empty?
```

Comment:
@alishaevn this needs to be a separate PR. You want to go to production with this still working and THEN fill in the tenants. once that is done, you can remove this as a different PR -> deploy step.

Reply:
ok, I'll move this to #482.

Reply:
moved!

Reply:
moved this to #487 instead.